client/app/scripts/components/node-details/node-details-table.js

```diff
@@ -169,10 +169,12 @@ export default class NodeDetailsTable extends React.Component {
           <tbody>
           {nodes && nodes.map(node => {
             const values = this.renderValues(node);
+            const nodeId = node.node_id || node.id;
```

Comment:
this is a connection-specific thing, so it doesn't really seem like something the node details table should know about.